Assets/Scripts/NetworkManager.cs

```diff
@@ -24,15 +24,11 @@ void Start()
                 {
                     PhotonNetwork.ConnectUsingSettings(GameConstants.VERSION_STRING);
                 }
-            }
-            else
-            {
-                Object.DontDestroyOnLoad(Instance);
-                Destroy(gameObject);
+                NetworkInitialized = true;

```

Comment:
Could NetworkInitialized be set to true within the if statement one line 14 and we do away with the else statement?